lms/djangoapps/instructor/views/api.py

```diff
@@ -343,6 +343,13 @@ def register_and_enroll_students(request, course_id):  # pylint: disable=too-man
     row_errors = []
     general_errors = []
 
+    # for White Labels we use 'shoppingcart' which is based on the
+    # "honor" course_mode.
```

Comment:
Hmm, I don't know that we can count on 'honor' always being the value of DEFAULT_SHOPPINGCART_MODE_SLUG, so we should instead describe why this particular conditional is important to the rest of the workflow.